src/main/java/com/pubnub/api/workers/SubscribeMessageWorker.java

```diff
@@ -68,6 +68,8 @@ private void takeMessage() {
             } catch (InterruptedException e) {
                 Thread.currentThread().interrupt();
                 log.trace("take message interrupted", e);
+            } catch (Exception e) { // don't crash the thread on malformed messages
```

Comment:
I am just wondering if this is needed anymore after changes in `processMessage` method ?
My understanding is that `processMessage` method handles malformed messages.

Reply:
yeah, I'd leave it in case there are errors in some other part of message processing, otherwise the whole message processing loop crashes

Reply:
I see.